### DISCUSSION TITLE
Reduce Tolerances using ospsuite R

### DISCUSSION THREAD

**Discussion by DR-99** [Open-Systems-Pharmacology/Forum#1589] [Questions & Problems] [answered]
Hi, I am running batch simulations with the R package and sometimes observe an error when performing a large number of simulations. Performing the simulations in R and PK-Sim do not produce errors, neither do small batches, and I am also sure that my input ranges are appropriate. I noticed that there has been discussion concerning this in the past, including "allowing negative values" via Mobi and a possible auto-reduce tolerance feature in the future, however I could not find information on reducing the tolerances within R. Is is possible to do so?

**Comment by Christoph27** [comment, marked as answer]
Yes, if your simulation is "sim" [sim = loadSimulation(simFilePath)]
then e.g.:
sim$solver$absTol = 1e-10 
sim$solver$relTol = 1e-7

**Reply by DR-99** [reply]
Thank you for the relevant commands! I tried reducing the tolerances and unfortunately it seemed to multiply the number of warnings for my batches. Would you happen to have any other tips?
